### PR TITLE
Jack flenna unittest

### PR DESCRIPTION
**Pull request by flennaja**

### CODE REVIEW

**Comment by flennaja** [comment]
> Why are there so many conflicting files in this branch??

@arushwolf Because it incorporates updates from the some-fixes branch that isn't merged into main yet. Looking to get that PR approved/merged within the next couple of days, and then once this branch is synced up with main again the only differences should be the unit tests

File: AppSettingsManagerAPI/AppSettingsManagerApi.Model/Requests/UpdatePermissionRequest.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace AppSettingsManagerApi.Model.Requests;
+
+public class UpdatePermissionRequest
+{
+    public string UserId { get; set; }
```
Comment (MsMarini) [inline AppSettingsManagerAPI/AppSettingsManagerApi.Model/Requests/UpdatePermissionRequest.cs:5]: overall looks like the same methods used as before, nicely done

File: AppSettingsManagerAPI/AppSettingsManagerApi.Tests/SettingFacadeTests.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    
+    private readonly SettingFacade _settingFacade;
+
+    public SettingFacadeTests()
+    {
+        _settingFacade = new SettingFacade(
+            _settingRepositoryMock.Object,
+            _permissionRepositoryMock.Object
+        );
+    }
```
Comment (MsMarini) [inline AppSettingsManagerAPI/AppSettingsManagerApi.Tests/SettingFacadeTests.cs:19]: Nice use of the Settings facade!

**Review by MsMarini — APPROVED** [review]
Overall looks good, we should merge so that those conflict errors go away ASAP

**Pull request closed without merging**